inspectify-api/Cargo.toml

```diff
@@ -17,6 +17,7 @@ chrono = { version = "0.4.33", features = ["serde"] }
 clap.workspace = true
 color-eyre = "0.6.2"
 driver.workspace = true
+dunce = { workspace = true }
```

Comment:
And here

Reply:
Yes, I've now addressed this in the latest commit 🙂